llpc/test/shaderdb/partial_pipeline_cache_test/pipelines/PipelineVsFs_test1_Vs1Fs1WithData.pipe

```diff
@@ -0,0 +1,106 @@
+; Test that constant data in the vertex shader is handled correctly.
+; BEGIN_SHADERTEST
+; RUN: amdllpc -spvgen-dir=%spvgendir% -enable-relocatable-shader-elf -o %t.elf %gfxip %s && llvm-objdump --arch=amdgcn --disassemble-zeroes --mcpu=gfx900 -D -r %t.elf | FileCheck -check-prefix=SHADERTEST %s
+; SHADERTEST-LABEL: <_amdgpu_vs_main>:
+; SHADERTEST: s_add_u32 {{s[0-9]*}}, {{s[0-9]*}}, 4
+; SHADERTEST-NEXT: R_AMDGPU_REL32_LO    [[vs_data_sym:[.a-z]*]]
+; SHADERTEST-NEXT: s_addc_u32 {{s[0-9]*}}, {{s[0-9]*}}, {{4|12}}
+; SHADERTEST-NEXT: R_AMDGPU_REL32_HI    [[vs_data_sym]]
+; SHADERTEST-LABEL: <_amdgpu_ps_main>:
+; SHADERTEST: s_add_u32 {{s[0-9]*}}, {{s[0-9]*}}, 4
+; SHADERTEST-NEXT: R_AMDGPU_REL32_LO    [[fs_data_sym:[.a-z]*]]
+; SHADERTEST-NEXT: s_addc_u32 {{s[0-9]*}}, {{s[0-9]*}}, {{4|12}}
+; SHADERTEST-NEXT: R_AMDGPU_REL32_HI    [[fs_data_sym]]
+; SHADERTEST: 0000000000000000 <[[vs_data_sym]]>:
+; SHADERTEST-NEXT: 000000000000: 3F800000
+; SHADERTEST-NEXT: 000000000004: 00000000
+; SHADERTEST-NEXT: 000000000008: 00000000
+; SHADERTEST-NEXT: 00000000000C: 3F800000
+; SHADERTEST-NEXT: 000000000010: 00000000
+; SHADERTEST-NEXT: 000000000014: 3F800000
+; SHADERTEST-NEXT: 000000000018: 00000000
+; SHADERTEST-NEXT: 00000000001C: 3F800000
+; SHADERTEST-NEXT: 000000000020: 00000000
+; SHADERTEST-NEXT: 000000000024: 00000000
+; SHADERTEST-NEXT: 000000000028: 3F800000
+; SHADERTEST-NEXT: 00000000002C: 3F800000
+; SHADERTEST: 0000000000000030 <[[fs_data_sym]]>:
+; SHADERTEST-NEXT: 000000000030: 3F800000
+; SHADERTEST-NEXT: 000000000034: 00000000
+; SHADERTEST-NEXT: 000000000038: 00000000
+; SHADERTEST-NEXT: 00000000003C: 3F800000
+; SHADERTEST-NEXT: 000000000040: 00000000
+; SHADERTEST-NEXT: 000000000044: 3F800000
+; SHADERTEST-NEXT: 000000000048: 00000000
+; SHADERTEST-NEXT: 00000000004C: 3F800000
+; SHADERTEST-NEXT: 000000000050: 00000000
+; SHADERTEST-NEXT: 000000000054: 00000000
+; SHADERTEST-NEXT: 000000000058: 3F800000
+; SHADERTEST-NEXT: 00000000005C: 3F800000
+; END_SHADERTEST
+
+; BEGIN_SHADERTEST
+; RUN: amdllpc -spvgen-dir=%spvgendir% -o %t.elf %gfxip %s && llvm-objdump --arch=amdgcn --disassemble-zeroes --mcpu=gfx900 -D -r %t.elf | FileCheck -check-prefix=SHADERTEST2 %s
+; SHADERTEST2-LABEL: <_amdgpu_vs_main>:
+; SHADERTEST2: s_add_u32 {{s[0-9]*}}, {{s[0-9]*}}, 4
+; SHADERTEST2-NEXT: R_AMDGPU_REL32_LO    .rodata
+; SHADERTEST2-NEXT: s_addc_u32 {{s[0-9]*}}, {{s[0-9]*}}, {{4|12}}
+; SHADERTEST2-NEXT: R_AMDGPU_REL32_HI    .rodata
+; SHADERTEST2-LABEL: <__llpc_global_proxy_>
+; SHADERTEST2-NEXT: {{[0-9]*}}: 3F800000
```

Comment:
If you expect at least one digit, `{{[0-9]+}}` should be better

Reply:
Replaced with the certain vaule.